src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,62 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private K[] arrayOfKeys;
+    private V[] arrayOfValues;
+    private int pointer = 0;
+
```

Comment:
there is no need to initialize primitive with it's default value
```suggestion
    private int pointer;

```